frontend/src/components/Recipe/ContextMenu.vue

```diff
@@ -123,10 +138,11 @@ export default {
     updateClipboard() {
       const copyText = this.recipeURL;
       navigator.clipboard.writeText(copyText).then(
-        () => console.log("Copied", copyText),
-        () => console.log("Copied Failed", copyText)
+        () => console.log("Copied to Clipboard", copyText),
+        () => console.log("Copied Failed", copyText),
+        utils.notify.success("Copied to Clipboard")
```

Comment:
Please check this again, got a warning:
Warning:(143, 9) Void function return value is used